sip/sip-007-stacking-consensus.md

```diff
@@ -579,6 +584,26 @@ This operation includes an `OP_RETURN` output for the first Bitcoin output:
 
 Where `op = x` (ascii encoded).
 
-Where the unsigned integers are big-endian encoded.
+Where the unsigned integer is big-endian encoded.
 
 The second Bitcoin output will be used as the reward address for any stacking rewards.
+
+### TransferStxOp
+
+The first input to the Bitcoin operation _must_ consume a UTXO that is
+the second output of a `PreStxOp`. This validates that the `TransferStxOp` was signed
+by the appropriate STX address.
+
+This operation includes an `OP_RETURN` output for the first Bitcoin output:
+
+```
+            0      2  3                             19        80
+            |------|--|-----------------------------|---------|
+             magic  op     uSTX to transfer (u128)     memo (up to 61 bytes)
+```
+
+Where `op = $` (ascii encoded).
+
+Where the unsigned integer is big-endian encoded.
+
+The second Bitcoin output is the B58-encoding of the recipient Stacks address.
```

Comment:
Bitcoin outputs do not contain addresses.  Should this say something about the `scriptPubKey` needing to be either a `p2pkh` or `p2sh` output, so that a Stacks address can be derived from the internal 20-byte hash?

Reply:
Added in https://github.com/blockstack/stacks-blockchain/pull/2129/commits/b4c7b8878ad442a81a5fe47b6e94de287de49305